docker/nginx/nginx.conf

```diff
@@ -42,6 +42,12 @@ server {
     }
 
     location ~ ^/resolve-friendly-url {
+        allow 10.0.0.0/8;
+        allow 127.0.0.0/8;
+        allow 172.0.0.0/8;
+        allow 192.0.0.0/8;
```

Comment:
Correct CIDR for those ranges are:

```
172.16.0.0/12
192.168.0.0/16
```

Furthermore, loopback IP (`127.0.0.0/8`) is useless, since requests never come from the nginx container itself